packages/synapse-react-client/src/utils/hooks/useUploadFileEntity/useUploadFileEntities.test.ts

```diff
@@ -27,22 +24,21 @@ import {
   InitiateUploadArgs,
   useUploadFileEntities,
 } from './useUploadFileEntities'
+import {
+  UploadItem,
+  useUploadFiles,
+  UseUploadFilesReturn,
+} from './useUploadFiles'
 
-vi.mock('../../../synapse-queries/file/useUploadDestination', () => {
+vi.mock('@/synapse-queries/file/useUploadDestination', () => {
   return {
     useGetDefaultUploadDestination: vi.fn(),
   }
 })
 
-vi.mock('../../../synapse-queries/file/useSynapseMultipartUpload', () => {
+vi.mock('./useUploadFiles', () => {
```

Comment:
Updated these tests to 

1. Mock the newly extracted `useUploadFiles` instead of the upload calls directly
2. Remove basic, non-entity related tests that are now captured in `useUploadFiles.test.ts`